modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -155,6 +158,35 @@ Ignore this field if you selected *none* as JWT Signing Method.
 Note that all DataWeave expressions must return a boolean value or they will always fail.
 |===
 
+== Obtaining a policy compatible PEM public key
+The policy uses public rsa key in pem format. In this section we explain how to obtain the public.pem file that contains said key.
```

Comment:
The policy uses a public RSA key in .pem format. This section describes how to obtain the public .pem file that contains the key.